tests/php/Forms/FormTest.php

```diff
@@ -1030,6 +1097,54 @@ public function testFieldMessageEscapeHtml()
         );
     }
 
+    /**
+     * @dataProvider formMessageDataProvider
+     */
+    public function testFieldMessageAppend($messages)
```

Comment:
```suggestion
    public function testFieldMessageAppend(array $message1, array $message2, bool $force = false)
```
Your data provider is passing in the arguments like this, which is why CI is unhappy. You could either refactor your dataprovider, or update this method to use this signature.
I think this ends up being tidier.